.nuget/Siemens.Simulation.SourceGrid.nuspec

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata>
+    <id>Siemens.Simulation.SourceGrid</id>
+    <version>4.40.1</version>
+    <title>Siemens.Simulation.SourceGrid</title>
+    <authors>AuthorName</authors>
```

Comment:
needs to fill proper author and owner, please contact Karthika to follow what is defined in CA